docs/tutorials/solar_flares.ipynb

```diff
@@ -903,8 +881,29 @@
     "Or remove the existing binning, yielding a 1D-variable.\n",
     "\n",
     "```python\n",
-    "temporal = sc.bin(spatial, edges=[time_edges], erase=['x', 'y'])\n",
-    "temporal.bins.sum().plot(ylabel='number of flares')\n",
+    "temporal = spatial.bins.concat('x').bins.concat('y').bin(peak_time=200)\n",
+    "temporal.hist().plot(ylabel='number of flares')\n",
+    "```\n",
+    "\n",
+    "<br>\n",
+    "Note that it is important to first remove the binning in x and y by using `concat`.\n",
+    "Binning in time first and then removing other binning would use too much memory.\n",
+    "It is possible to do everything in one step using a lower-level API:\n",
+    "\n",
+    "```python\n",
+    "from scipp.binning import make_binned\n",
+    "\n",
+    "time = spatial.bins.coords['peak_time']\n",
+    "min_time = time.min().value\n",
+    "max_time = time.max().value\n",
+    "step = (max_time - min_time) / 200\n",
+    "time_edges = sc.arange('peak_time',\n",
```

Comment:
I think using `linspace` here would simplify the code a little?

Reply:
Would be nice, but linspace does not support datetimes (limitation in numpy).

Reply:
Ok. Does our linspace support datetime? If not can we make it so?

Reply:
It doesn't because it uses `np.linspace`. I think the reason it is not supported is that you can't guarantee equal spacing because datetimes are necessarily integers. If you call linspace with ints, you get an array of floats...